src/python/foglamp/storage/storage.py

```diff
@@ -104,35 +104,13 @@ def check_service_availibility(self):
         conn.close()
         return json.loads(res)
 
-    # TODO: remove me, and allow this call in service registry API
-    def shutdown(self):
-        """ stop Storage service """
-
-        conn = http.client.HTTPConnection(self.management_api_url)
-        # TODO: need to set http / https based on service protocol
-
-        conn.request('POST', url='/foglamp/service/shutdown', body=None)
-        r = conn.getresponse()
-
-        # TODO: FOGL-615
-        # log error with message if status is 4xx or 5xx
-        if r.status in range(400, 500):
-            _LOGGER.error("Client error code: %d", r.status)
-        if r.status in range(500, 600):
-            _LOGGER.error("Server error code: %d", r.status)
-
-        res = r.read().decode()
-        conn.close()
-        return json.loads(res)
-
-    def _get_storage_service(self):
+    def _get_storage_service(self, host, port):
         """ get Storage service """
 
-        # TODO: URL to service registry api?
-        conn = http.client.HTTPConnection("{0}:{1}".format("", ""))
+        conn = http.client.HTTPConnection("{0}:{1}".format(host, port))
         # TODO: need to set http / https based on service protocol
 
-        conn.request('GET', url='/foglamp/service')
+        conn.request('GET', url='/foglamp/service?name=FogLAMP%20Storage')
```

Comment:
Is this correct?

Reply:
Yes, for service discovery by name. for %20 we can do explicity encoding. but I believe its not needed. as name is fixed.